airflow/models/dagrun.py

```diff
@@ -650,7 +650,6 @@ def get_task_instance(
         )
 
     @staticmethod
-    @internal_api_call
```

Comment:
this is a drive-by.... it doesn't need to be RPC AFAICT

Reply:
cc @vincbeck

Reply:
Basis for conclusion is, I can run task without it.